<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>379</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI Semilight</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/3dadf139-3c6e-4a2b-90df-edf28f83b81b.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #cf617a; /* Розовый цвет фона */
    color: white; /* Белый цвет текста */
    border: none; /* Убрать рамку */
    border-radius: 20px; /* Закругление углов для овала */
     padding: 10px 20px; /* Внутренние отступы */
    font-size: 16px; /* Размер шрифта */
    /* Задаем овальную форму */
}

QPushButton:hover {
    background-color: #c36077; /* Цвет фона при наведении */
}

QPushButton:pressed {
    background-color: #703845; /* Цвет фона при нажатии */
}</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_login">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#lineEdit {
    border-radius: 200px;
    border: 1px solid transparent; /* Добавляем прозрачную рамку для работы маски */
    border-image: url(&quot;&quot;) 1 1 1 1 stretch stretch; /* Устанавливаем border-image */
    outline: none;
}
 QLineEdit#lineEdit:focus {
  border-image: none; /* Убираем border-image при фокусе */

}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#lineEdit {
    border-radius: 200px;
    border: 1px solid transparent; /* Добавляем прозрачную рамку для работы маски */
    border-image: url(&quot;&quot;) 1 1 1 1 stretch stretch; /* Устанавливаем border-image */
    outline: none;
}
 QLineEdit#lineEdit:focus {
  border-image: none; /* Убираем border-image при фокусе */

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>150</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/Telegram Desktop/yfv yv.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/Telegram Desktop/123mgnv.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/книги.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
